app/models/episode.rb

```diff
@@ -154,10 +164,15 @@ def file_size
   def copy_media(force = false)
     enclosures.each { |e| e.copy_media(force) }
     all_contents.each { |c| c.copy_media(force) }
+    images.each{ |i| i.copy_media(force) }
   end
 
   def base_published_url
-    "#{podcast.base_published_url}/#{guid}"
+    "https://#{feeder_cdn_host}/#{path}"
```

Comment:
getting to it differently - I now need the podcast + episode path, and before the code was parsing the final url to get it, now I added a `path` method to get it from the episode, and used it here as well.